audio_generation/distil-whisper/main.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+from openvino.runtime import Core
+import torch
+from datasets import load_dataset
+from transformers import AutoProcessor
+from optimum.intel.openvino import OVModelForSpeechSeq2Seq
+
+#Define model parameters
+distil_model_id = "distil-whisper/distil-large-v2"
+core = Core()
+device = "GPU"
+processor = AutoProcessor.from_pretrained(distil_model_id)
+
+#Specify model dir
+model_dir = Path(distil_model_id.split("/")[-1])
+
+#Prepare and load dataset
+def extract_input_features(sample):
```

Comment:
This example is fully based on Optimum-Intel. @riacheruvu, don't you think that it should be a part of this [documentation](https://huggingface.co/docs/optimum/intel/inference)?